pom.xml

```diff
@@ -71,8 +71,8 @@
     <version.io.github.openfeign.opentracing>0.0.5</version.io.github.openfeign.opentracing>
     <!-- spring-boot-starter-parent is a module of spring-boot-dependencies
         https://github.com/spring-projects/spring-boot/blob/master/spring-boot-starters/spring-boot-starter-parent/pom.xml -->
-    <version.org.springframework.boot>1.4.1.RELEASE</version.org.springframework.boot>
-    <version.org.springframework.cloud-spring-cloud-dependencies>Camden.SR7</version.org.springframework.cloud-spring-cloud-dependencies>
+    <version.org.springframework.boot>1.5.8.RELEASE</version.org.springframework.boot>
```

Comment:
https://start.spring.io/ for spring 1.5.8.RELEASE uses Dalston.SR4. 
http://projects.spring.io/spring-cloud/ for 1.5.6.RELEASE uses Dalston.SR4. 

is 1.5.8.RELEASE with Dalston.SR3 supported by RHOAR?

Reply:
Spring Boot 1.5.8.RELEASE yes. Dalston.SR3 not exactly. RHOAR bom includes only a few pieces of Spring Cloud: https://github.com/snowdrop/spring-boot-bom/blob/1.5.8.Final/pom.xml#L175. However, they're not exactly aligned with neither Dalston.SR3 nor Dalston.SR2. I think Kamesh added them, so will have to check with him once he's back and update them. However, Dalston.SR3 is the one which includes the correct version of Spring Cloud Stream used by Artemis binder.

Reply:
Then let's go with SR3 for now. I am not sure if there are any compatibility differences though.

Reply:
Yes, the missmatch is only in the micro version. There is a much bigger mismatch with Camden.SR7.

Reply:
I've just checked again and to be honnest both SR3 and SR4 have mismatches with spring-boot-bom, howeve all the differences are with micro versions. So it should be ok to use Dalston.SR4 and I will mention in my team to update versions according to that for the next iteration.

Reply:
+1 I think SR4 is better as spring refers to it with `1.5.8.RELEASE`